### DISCUSSION TITLE
show the shortcuts app generated by the browser

### DISCUSSION THREAD

**Discussion by ahmed-n-abdeltwab** [tanujnotes/Olauncher#298] [Ideas]
it's would be a good idea to add this feature.

**Comment by Link2324** [comment]
I was also thinking about this. is anyone already working on it?

**Reply by tanujnotes** [reply]
I plan to work on this. But it's going to take some time.

**Reply by Link2324** [reply]
How much time? Before that can i try to do it?

**Reply by tanujnotes** [reply]
Yes please go ahead.

**Reply by eXaLy** [reply]
@Link2324 @ahmed-n-abdeltwab 
See https://github.com/tanujnotes/Olauncher/issues/20 and https://github.com/tanujnotes/Olauncher/pull/682 😄

**Reply by ahmed-n-abdeltwab** [reply]
congrats
